sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Integration.Tests/Azure.Monitor.OpenTelemetry.Exporter.Integration.Tests.csproj

```diff
@@ -26,12 +26,12 @@
     <PackageReference Include="Microsoft.AspNetCore.Mvc" VersionOverride="[2.1.1,2.2)" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.1'">
-    <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" VersionOverride="3.1.0" />
+  <ItemGroup Condition="'$(TargetFramework)' == 'net6.0'">
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" VersionOverride="5.0.0" />
```

Comment:
Curious, why use 5.0.0 instead of 6.0.0 for net6.0?

Reply:
Contrary to what GitHub shows, that part hasn't changed. :)
5.0.0 must have been the minimum version that worked when we created this project. I thought about bumping it to 6.0.0, but decided to leave it as is for this PR.

Reply:
Is it the normal practice to use an older testing framework on a newer version of runtime?

Reply:
we should update this.

Reply:
good point. 
i left it out to focus on re-enabling the tests. but it's a small fix and i can include it here. thanks